sdk/connect/javascript-wagmi.md

```diff
@@ -11,9 +11,9 @@ Get started with MetaMask SDK in a JavaScript and Wagmi dapp.
 You can [download the quickstart template](#set-up-using-a-template) or [manually set up the SDK](#set-up-manually) in an existing dapp.
 
 <p align="center">
-  <a href="https://metamask-wagmi-demo.vercel.app/" target="_blank">
-    <img src={require("../_assets/quickstart.jpg").default} alt="Quickstart" width="600px" />
-  </a>
+  <!-- a href="https://metamask-wagmi-demo.vercel.app/" target="_blank" -->
+    <img src={require("../_assets/quickstart.jpg").default} alt="Quickstart" width="600px" class="appScreen" />
+  <!-- /a -->
```

Comment:
Commented out this link because I get this error when trying to reach it. Is there an easy fix?
<img width="1176" height="1113" alt="Screenshot 2025-10-30 at 10 53 18 AM" src="https://github.com/user-attachments/assets/1c002698-d025-4762-8f8b-e6b5eb916df9" />